Gemfile.lock

```diff
@@ -261,14 +261,14 @@ GEM
       rspec-mocks (~> 3.4.0)
       rspec-support (~> 3.4.0)
     rspec-support (3.4.1)
-    rubocop (0.37.2)
-      parser (>= 2.3.0.4, < 3.0)
+    rubocop (0.40.0)
```

Comment:
@chriskacerguis (I'm glad you're interested in this)
The server is running ruby 2.2.4
Unfortunately the steps are:
  - ssh to the box
  - upgrade ruby to 2.2.5
  - notice jobs going to retry because of Parser errors wanting 2.2.4
  - switch back to 2.2.4